### DISCUSSION TITLE
Sep. 26 Team Check In

### DISCUSSION THREAD

**Discussion by umarbilgrammi** [ecn310/course-project-taxes-tariffs#4] [General]
### **09/26/2024**
– Something someone did to help the team this week: I helped Dylan by giving him some advice for an interview he had for a club on campus. 
– Something I could have done better for the team this week: I could have communicated better that I was going to miss class on Tuesday. 
– An area/suggestion for team improvement: I think our team is doing well and we will adjust as needed. We all get along well so there should be no outstanding issues :)

**Comment by DylanTh20** [comment]
### **09/26/2024**
– Something someone did to help the team this week: I helped Umar by completing the class log on a day he was supposed to complete it because he was absent.
– Something I could have done better for the team this week: I could have communicated better while we were working on our second paper summary.
– An area/suggestion for team improvement: I think our team is doing well and we will continue to work effectively. When we start meeting in person I am confident we will be able to work through any issues quite easily.

**Comment by Ksarrge** [comment]
## 09/26/2024
- Something someone did to help the team this week: Dylan presented how we got our data to the class.
- Something I could have done better: Helped with the first paper summary better
- Any suggestions: I think we are doing pretty solid currently. I have had no issues.
